modules/jupyterhub-dask-eks/input.tf

```diff
@@ -53,14 +53,13 @@ variable "vpc_id" {
 variable "kubernetes_version" {
   description = "Kubernetes Cluster version"
   type        = string
-  default     = "1.25"
+  default     = "1.29"
 }
 
 variable "kubernetes_autoscaler_version" {
   description = "Kubernetes Cluster Autoscaler version"
   type        = string
-  default     = "v1.25.0"
-  # note: v1.25.3 seems not to work correctly, see FD-1594
+  default     = "v1.29.0"
```

Comment:
is there a 1.29.0 of this?

(I don't know if i'm looking in the right place, but: https://github.com/kubernetes/autoscaler/releases)

Reply:
Yes, there is: https://github.com/kubernetes/autoscaler/releases/tag/cluster-autoscaler-1.29.0